docs/admin/collecting-metrics/serving-metrics/metrics.md

```diff
@@ -33,6 +33,7 @@ Autoscaler component exposes a number of metrics related to its decisions per re
 | not_ready_pods | Number of pods that are not ready currently | Gauge | configuration_name=<br>namespace_name=<br>revision_name<br>service_name |  Dimensionless | Stable |
 | pending_pods | Number of pods that are pending currently | Gauge | configuration_name<br>namespace_name<br>revision_name<br>service_name | Dimensionless | Stable |
 | terminating_pods | Number of pods that are terminating currently | Gauge | configuration_name<br>namespace_name<br>revision_name<br>service_name<br> | Dimensionless | Stable |
+| scrape_time | Time to scrape metrics from the service pods in milliseconds | Histogram | configuration_name<br>namespace_name<br>revision_name<br>service_name<br> | Milliseconds | Stable |
```

Comment:
This could be slightly clearer. Is this the time autoscaler takes to scrape the metrics?

Reply:
Yes.

Reply:
```suggestion
| scrape_time | Time autoscaler takes to scrape metrics from the service pods in milliseconds | Histogram | configuration_name<br>namespace_name<br>revision_name<br>service_name<br> | Milliseconds | Stable |
```